src/middlewared/middlewared/utils/os.py

```diff
@@ -42,12 +42,17 @@ def close_fds(low_fd, max_fd=None):
     closerange(low_fd, max_fd)
 
 
-def terminate_pid(pid: int, timeout: int = 10) -> bool:
-    # Send SIGTERM to request the process to terminate
-    kill(pid, SIGTERM)
+def terminate_pid(pid: int, timeout: int = 10, get_pgid: bool = False) -> bool:
```

Comment:
Should it be named `use_pgid`? `get` sounds like we’ll return something. Also, this method now deserves a params doc.

Reply:
Sure, fixed it and added docstring.